invenio_communities/templates/semantic-ui/invenio_communities/macros/featured_communities.html

```diff
@@ -0,0 +1,23 @@
+{# -*- coding: utf-8 -*-
+
+    This file is part of Invenio.
+    Copyright (C) 2022 CERN.
+
+    Invenio is free software; you can redistribute it and/or modify it
+    under the terms of the MIT License; see LICENSE file for more details.
+#}
+
+{% macro featured_communities(
+  column_width=4,
```

Comment:
what happens if the number of communities featured in the system is lower than these numbers?

Reply:
They will be centered in the middle of the page as in the following screenshot:
![screencapture-127-0-0-1-5000-2022-11-10-08_52_28](https://user-images.githubusercontent.com/15194802/201055422-075300e7-acc1-443a-8ab8-971bc0dca901.png)